src/super_gradients/recipes/quantization_params/default_tensorrt_ptq.yaml

```diff
@@ -0,0 +1,34 @@
+quantizer:
+  TRTPTQQuantizer:
+    selective_quantizer_params:
+      calibrator_w: "max"        # calibrator type for weights, acceptable types are ["max", "histogram"]
+      calibrator_i: "histogram"  # calibrator type for inputs acceptable types are ["max", "histogram"]
+      per_channel: True          # per-channel quantization of weights, activations stay per-tensor by default
+      learn_amax: False          # enable learnable amax in all TensorQuantizers using straight-through estimator
+      skip_modules: []
+
+    calib_params:
+      histogram_calib_method: "percentile"  # calibration method for all "histogram" calibrators, acceptable types are ["percentile", "entropy", "mse"], "max" calibrators always use "max"
+      percentile: 99.99                     # percentile for all histogram calibrators with method "percentile", other calibrators are not affected
+      num_calib_batches: 128                # number of batches to use for calibration, if None, 512 / batch_size will be used
+      verbose: False                        # if calibrator should be verbose
+
+exporter: ONNXExporter
+
+output_path: ${architecture}_trt_ptq_int8.onnx
+
+# Model-specific parameters
+export_params:
```

Comment:
We need to properly document all the export params with their corresponding exporters.
I would feel a bit lost if I wanted lets say, try and build a TRT engine on a segmentation model for instance.